Backpack/src/main/java/net/skyscanner/backpack/card/BpkCard.kt

```diff
@@ -0,0 +1,51 @@
+package net.skyscanner.backpack.card
+
+import android.content.Context
+import android.support.annotation.Dimension
+import android.support.v7.widget.CardView
+import android.util.AttributeSet
+import net.skyscanner.backpack.R
+
+
+open class BpkCard @JvmOverloads constructor(
```

Comment:
General comment about naming: Why don't we call these components as `*View` - thinking here of `BpkText`, `BpkCard`, etc.

Reply:
On iOS the `BpkText` is named `BpkTextView` but the rest is not - again inconsistency

![image](https://user-images.githubusercontent.com/935897/46603297-f5d3e080-caf2-11e8-908d-0b7166c42399.png)